jupyterlab_git/git.py

```diff
@@ -193,7 +193,7 @@ def __init__(self, contents_manager):
                 response["command"] = " ".join(cmd)
                 response["message"] = error
             else:
-                response["files"] = output.strip().split("\n")
+                response["files"] = output.strip("\x00").split("\x00")
```

Comment:
given you call `.strip("\x00").split("\x00")` three times in this change it might be worth creating function for it.